src/cmd/service-name/main.go.tmpl

```diff
@@ -0,0 +1,37 @@
+package main
```

Comment:
does github have file comments?

Anyway, the directory is still called "service-name". In the generator-main.yaml the target path is evaluated as a text/template too, so you can just use the actual service name.

Or maybe since we don't actually have more than one cmd in our services (long term, they're intended to be a container just containing one static binary and possibly one openapi spec file), maybe drop the service-name directory altogether and move directly under cmd.

Reply:
- [ ] Remove service subdirectory